pkg/network/ovn_kubernetes.go

```diff
@@ -108,7 +108,22 @@ func renderOVNKubernetes(conf *operv1.NetworkSpec, bootstrapResult *bootstrap.Bo
 		data.Data["OVNHybridOverlayVXLANPort"] = ""
 	}
 
+	if c.IPsecConfig != nil {
+		data.Data["EnableIPsec"] = c.IPsecConfig.Enable
```

Comment:
Nit: we don't really need to set `data.Data["EnableIPsec"]` right? The ovn-kubernetes IP sec daemonset will be rendered based on what happens in this function, the template variable has no impact, am I right?

Reply:
I can remove that now. The last change I made removed templating of the ovnkube-master yaml (which was needed) but I didnt think of removing this.

Reply:
I'll keep it in because I made a change to that needs this now!